docs/submitting_data.rst

```diff
@@ -168,7 +168,7 @@ data structures to send, do something like this.
 
 .. code-block:: python
 
-    from thclient import (TreeherderAuth, TreeherderClient, TreeherderClientError,
+    from thclient import (HawkAuth, TreeherderClient, TreeherderClientError,
```

Comment:
> I was thinking we should provide a slim wrapper around it in the client's auth.py, to reduce boilerplate, along the lines of:
> https://bugzilla.mozilla.org/show_bug.cgi?id=1209555#c5
> 
> eg like:
> https://github.com/mozilla/treeherder/compare/hawkauth-wrapper

Mauro suggested I see about improving this upstream, so I've filed https://github.com/mozilla-services/requests-hawk/issues/8